src/agent_factory/utils/artifact_validation.py

```diff
@@ -30,6 +30,8 @@ def validate_dependencies(tools: str, dependencies: list[str]) -> str:
       specifiers) is replaced with exactly `litellm<1.75.0`.
     - If no `litellm` line exists and dependencies are non-empty, append
       `litellm<1.75.0` as a new line.
+    - if visit_webpage is in tools, add markdownify==1.2.0 to dependencies
```

Comment:
Could you add a comment documenting why we need to pin `markdownify` and `tavily-python`, for future references?